pbparam/optimisation_problems/data_fit.py

```diff
@@ -135,15 +144,15 @@ def setup_cost_function(self):
             The value of the cost function evaluated at x.
         """
         simulation = copy.deepcopy(self.simulation)
-        cost_function = partial(
+        cost_func = partial(
             cost_function_full,
             simulation,
             self.map_inputs,
             self.scalings,
             self.data,
             self.variables_optimise,
         )
-        self.cost_function = cost_function
+        self.cost_func = cost_func
```

Comment:
It is better to keep it as `cost_function` instead of `cost_func`

Reply:
Would it not be confusing for cost_function class and cost_function with partial?

Reply:
Oh yes, good point. We now have the cost function (as RMSE, MLE...) and the function we are minimising... Shall we rename the latter, internally, to `objective_function`?

Reply:
Yes. Actually that's a better name.